worldcoin/tech-blog
# Add redirects to the company website

Related to [this](https://ottofeller.slack.com/archives/C048NGG1NK0/p1675508164561479?thread_ts=1675438542.295089&cid=C048NGG1NK0) message

- Rename `4844-testimonial` blog page to `eip-4844`
- Add redirect from `/` to `https://worldcoin.org/blog/developers`
- Add redirect from `/blog/eip-4844` to `https://worldcoin.org/blog/developers/blog/eip-4844`
  - Not sure about this one. It should be `blog/4844-testimonial` -> `https://worldcoin.org/blog/developers/4844-testimonial` relying on slack message, but we renamed this doc.
